tests/attested_tls/oe_enclave/host/makefile

```diff
@@ -0,0 +1,38 @@
+# Copyright Microsoft. 
+# Licensed under the attached Microsoft Software License Terms
+
+# Detect C and C++ compiler options
+# if not gcc and g++, default to clang-7
+C_COMPILER=$(notdir $(CC))
+ifeq ($(C_COMPILER), gcc)
+        CXX_COMPILER=$(notdir $(CXX))
+        USE_GCC = true
+endif
+
+ifeq ($(USE_GCC),)
+        CC = clang-7
+        CXX = clang++-7
+        C_COMPILER=clang
+        CXX_COMPILER=clang++
+endif
+
+CFLAGS=$(shell pkg-config oehost-$(C_COMPILER) --cflags)
+CXXFLAGS=$(shell pkg-config oehost-$(CXX_COMPILER) --cflags)
+LDFLAGS=$(shell pkg-config oehost-$(CXX_COMPILER) --libs)
+INCDIR=$(shell pkg-config oeenclave-$(C_COMPILER) --variable=includedir)
+COMMONS="../../common"
+
+all: build
+
+build:
+	@ echo "Compilers used: $(CC), $(CXX)"
+	oeedger8r ../tlssrv.edl --untrusted \
+		--search-path $(INCDIR) \
+		--search-path $(INCDIR)/openenclave/edl/sgx
+	$(CC) -c $(CFLAGS) $(INCLUDES) tlssrv_u.c
+	$(CC) -c $(CXXFLAGS) $(INCLUDES) $(COMMONS)/settings.c
+	$(CXX) -c $(CXXFLAGS) $(INCLUDES) host.cpp
+	$(CXX) -o tlssrv_host host.o tlssrv_u.o settings.o $(LDFLAGS) -lpthread -lcurl -lmbedtls -lmbedcrypto
```

Comment:
You need to add -L/opt/openenclave/lib/openenclave/enclave after $(LDFLAGS) , otherwise you get a build error saying cannot find lmbedtls and lmbedcrypto